web-app/js/portal/search/FacetedSearchResultsDataView.js

```diff
@@ -57,6 +57,10 @@ Portal.search.FacetedSearchResultsDataView = Ext.extend(Ext.DataView, {
                 },
                 getIconUrl: function(values) {
                     return Ext.ux.Ajax.constructProxyUrl(this.getGeonetworkImageUrl(values.iconSourceUuid));
+                },
+                getTrimmedTitle: function(values) {
+                    var title = values.title;
+                    return String.format("<h3 title=\"{0}\">{1}</h3>", title, Ext.util.Format.ellipsis(title, 180, true));
```

Comment:
My thinking was that the adding of the title attribute to the html was best done in the function 'getTrimmedTitle'. Initially I thought 'getTrimmedTitle' would call a global function and the heading size (H3) would be an parameter to that global function, but that was going too far.
So is there a good reason to change it too your sugestion?